src/frontend/js/components/PaymentButton/index.tsx

```diff
@@ -227,7 +261,7 @@ const PaymentButton = ({ product, billingAddress, creditCard, onSuccess }: Payme
     <div className="payment-button">
       <Button
         disabled={state === ComponentStates.LOADING}
```

Comment:
may be
 ```js
   disabled={!readyToPay  || state === ComponentStates.LOADING}
 ```

Reply:
I agree, but it was like this before, because I believe it's to display an error message if information is missing